Makefile

```diff
@@ -8,6 +8,11 @@ all: build test
 
 clean:
 	go clean ./...
+	cd ../kubo && git co -- go.* plugin/*
```

Comment:
When deps changes take place (e.g. updating to diff version of go-car-mirror) you need to clean out go.* files in kubo as well.